server/apiserver/argoserver.go

```diff
@@ -116,9 +115,16 @@ func init() {
 	}
 }
 
+func getSSONamespace(opts ArgoServerOpts) string {
+	if opts.ManagedNamespace != "" {
```

Comment:
replace with `opts.ManagedNamespace` and then inline

Reply:
so `opts.ManagedNamespace` will be set to `""` if its not a namespaced installation
In that case `SSONamespace` will be `opts.Namespace`

Reply:
Ah - I need

Reply:
Minor - but I would have had `opts.GetSSONamespace()`

Reply:
I see, 
then I'll just resolve SSONamespace and pass it in opts.
and get as `opts.ssoNamespace`

Reply:
done